pkg/tsdb/loki/frame.go

```diff
@@ -0,0 +1,121 @@
+package loki
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+// we adjust the dataframes to be the way frontend & alerting
+// wants them.
+func adjustFrame(frame *data.Frame, query *lokiQuery) *data.Frame {
+	labels := getFrameLabels(frame)
+
+	timeFields, nonTimeFields := partitionFields(frame)
+
+	isMetricFrame := nonTimeFields[0].Type() != data.FieldTypeString
+
+	isMetricRanged := isMetricFrame && query.QueryType == QueryTypeRange
```

Comment:
Is `isMetricRanged` on purpose or should it be `isMetricRange`

Reply:
fixed